test/common.js

```diff
@@ -57,6 +57,20 @@ exports.validate = (t, report, options) => {
                 new RegExp('Node.js version: ' + process.version),
                 'NodeReport section contains expected Node.js version');
       }
+      if (options && options.commandline) {
+        if (this.isWindows()) {
+          // On Windows we need to strip double quotes from the command line in
+          // the report, and escape backslashes in the regex comparison string.
```

Comment:
It might depend on the shell or the executable path. This is what I am seeing on Windows:
```
==== NodeReport ================================================================

Event: JavaScript API, location: "TriggerReport"
Filename: NodeReport.20170110.113318.2184.001.txt
Dump event time:  2017/01/10 11:33:18
Module load time: 2017/01/10 11:33:18
Process ID: 2184
Command line: "C:\Program Files\nodejs6\node.exe" C:\tmp\nodereport\test\test-api-bad-processobj.js child
```